src/Pages/Detail/DetailNav.js

```diff
@@ -0,0 +1,40 @@
+import React, { Component } from "react";
+import "./DetailNav.scss";
+
+const NAV_ID = [
+  { id: "공간소개" },
+  { id: "시설안내" },
+  { id: "유의사항" },
+  { id: "환불정책" },
+  { id: "이용후기" },
+];
```

Comment:
이거는 지금도 잘 하셨는데 참고만 해주세요!
```suggestion
const NAV_ID = ["공간소개", "시설안내", "유의사항", "환불정책", "이용후기"];
```

Reply:
네!